tasks/system/RedHat.yml

```diff
@@ -13,9 +13,6 @@
 
     - name: "Install requirements"
```

Comment:
duplicate of tasks/system/Debian.yml. Is it Ok?

Reply:
it is in block